ebos/eclgenericoutputblackoilmodule.cc

```diff
@@ -1075,6 +1150,104 @@ doAllocBuffers(unsigned bufferSize,
         rstKeywords["BG"] = 0;
         invB_[gasPhaseIdx].resize(bufferSize, 0.0);
     }
+    
+    enableFlows_ = false;
+    enableFlowsn_ = false;
+    if (rstKeywords["FLOWS"] > 0) {
+        rstKeywords["FLOWS"] = 0;
+        enableFlows_ = true;
+        if (FluidSystem::phaseIsActive(gasPhaseIdx)){
```

Comment:
Instead of doing identical logic with three different indices you could:
```c++
auto phaseIdxs = std::array{ gasPhaseIdx, oilPhaseIdx, waterPhaseIdx }; // Using class template argument deduction to avoid saying the template arguments for array, neat!
auto compIdxs = std::array{gasCompIdx, oilCompIdx, waterCompIdx };
for (int ii = 0; ii < phaseIdxs.size(); ++i) {
    if (FluidSystem::phaseIsActive(phaseIdxs[ii]) {
    .... Use compIdx etc.
    }
}